shell/platform/embedder/embedder.h

```diff
@@ -707,6 +710,29 @@ FlutterEngineResult FlutterEngineSendPlatformMessage(
     FlutterEngine engine,
     const FlutterPlatformMessage* message);
 
+// Create a platform message response handle that allows the embedder to set a
+// native callback for a response to a message. This handle may be set on the
+// |response_handle| field of any |FlutterPlatformMessage|. The handle must be
+// collected via a call to |FlutterPlatformMessageReleaseResponseHandle|. This
+// may be done immediately after a call to |FlutterEngineSendPlatformMessage|
+// with a platform message whose response handle contains the handle created
+// using this call. In case a handle is created but never sent in a message, the
+// release call must still be made. Not calling release on the handle results in
+// a small memory leak.
+FLUTTER_EXPORT
+FlutterEngineResult FlutterPlatformMessageCreateResponseHandle(
+    FlutterEngine engine,
+    FlutterDataCallback data_callback,
+    void* user_data,
+    FlutterPlatformMessageResponseHandle** response_out);
+
+// Collect the handle created using
```

Comment:
s/Collect/Collects/ (if this function is kept)

Reply:
Done.